frontend/src/components/LoginComponent.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import { Button, Row, Col, Label } from "reactstrap";
+import StudentComponent from "./StudentComponent";
+import { Control, LocalForm, Errors } from "react-redux-form";
+import { Switch, Route, Redirect } from "react-router-dom";
+import AdminComponent from "./AdminComponent";
+import { withRouter } from "react-router-dom";
+
+const required = (val) => val && val.length;
+
+function LoginComponent(props) {
+  const [login, setLogin] = useState({
+    approved: true,
+    type: "admin"
+  });
+  const [userData, setUserData] = useState({});
+
+  const handleSubmit = (values) => {
+    if (login.type === "admin" && login.approved === true){
+      props.history.push("/admin");
+    } 
+    else if (login.type === "student" && login.approved === true){
+      
+      props.history.push("/student");
+    } else{
+      alert('Incorrect Username or Password Entered')
+    }
+
+    /*setLogin({
+      ...login,
+      approved: true,
+      type:"admin",
+      userdata: {}
+    });
+    alert(JSON.stringify(login));
+    alert(JSON.stringify(values));
+    */
+    
+    fetch("http://localhost:3000/users/login", {
+      method: "post",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        username: values.username,
+        password: values.password,
+      }),
+      credentials: 'same-origin'
+    })
+      .then((response) => response.json())
+      .then((item) => {
+        //checkType(item);
+        alert(JSON.stringify(item));
+        console.log(JSON.stringify(item))
+      })
+      .catch((err) => alert(err));
+  };
+
+  /*const checkType = (item) => {
```

Comment:
please remove unnecessary comments

Reply:
Even this part will be used once a proper response from the backend is received